test/sigunp.spec.js

```diff
@@ -1,5 +1,5 @@
 const supertest = require('supertest');
-const { expect } = require('chai');
+const { expect: expectChai } = require('chai');
```

Comment:
Why do you rename this to `expectChai`?

Reply:
I initially started the email tests in the signup suit. But when I run into some issues I created a suite for email testing (I will reintegrated it here though) and forgot to remove it. So.. to check that the email function has been called I'm using jest's expect, so I renamed chai's expect to keep using both. I did not realize initially I could simply use jest's expect everywhere.

Reply:
I realized now that you are using chai expect in test suites, you should use jest integrated expect (change it only in this new test suit or in signup only)